.pre-commit-config.yaml

```
@@ -56,3 +56,10 @@ repos:
     rev: v2.11
     hooks:
       - id: vulture
+  - repo: local
+    hooks:
+      - id: generate-manpages
+        name: Generate manpages
+        entry: podman run --rm Containerfiles/manpage.Containerfile
```

Did you test this locally? It doesn't work

```
convert2rhel on  pr/Andrew-ang9/1106:updated_fix_manpage [$] via 🐍 v3.12.3 
❯ podman run --rm Containerfiles/manpage.Containerfile          
Error: repository name must be lowercase
```
You could probably do this instead. Where `podman build`'s `-q` is quiet and will only output the image hash, which will then execute to run as `podman run --rm <image_hash>`
```suggestion
        entry: podman run --rm $(podman build -q -f Containerfiles/manpage.Containerfile)
```

---

Yeah I have done some testing locally and ran into some issues. I asked for the review anyways to see if I could get a little help with this. Thank you!